appserver/tests/tck/cdi/cdi-model/pom.xml

```diff
@@ -129,7 +129,7 @@
                     <reuseForks>true</reuseForks>
                     <!-- System Properties -->
                     <systemPropertyVariables>
-                        <glassfish.home>${glassfish.root}/glassfish8</glassfish.home>
+                        <glassfish.home>${glassfish.root}/glassfish7</glassfish.home>
```

Comment:
Is `glassfish7` correct folder?

Reply:
I should have mentioned; didn't do / try the other two yet besides API. Just did global replace for the version.